external/storm-eventhubs/src/main/java/org/apache/storm/eventhubs/spout/EventHubReceiverImpl.java

```diff
@@ -65,77 +60,80 @@ public EventHubReceiverImpl(EventHubSpoutConfig config, String partitionId) {
   }
 
   @Override
-  public void open(IEventHubFilter filter) throws EventHubException {
-    logger.info("creating eventhub receiver: partitionId=" + partitionId + 
-    		", filterString=" + filter.getFilterString());
+  public void open(String offset) throws EventHubException {
+    logger.info("creating eventhub receiver: partitionId=" + partitionId +
+            ", offset=" + offset);
     long start = System.currentTimeMillis();
-    receiver = new ResilientEventHubReceiver(connectionString, entityName,
-    		partitionId, consumerGroupName, defaultCredits, filter);
-    receiver.initialize();
-    
+    try {
+      ehClient = EventHubClient.createFromConnectionStringSync(connectionString);
+      receiver = ehClient.createEpochReceiverSync(
+              consumerGroupName,
+              partitionId,
+              offset,
+              false,
+              1);
```

Comment:
Please refrain from using catch all exceptions and move to catching more specific exceptions...